packages/algorand-js/src/runtime/runtime.ts

```diff
@@ -342,17 +356,19 @@ export class Runtime {
    * @param accounts : accounts passed by user
```

Comment:
there is no `accounts` parameter in this function. Could you update the function description? Please be more specific which account balances are we updating - it seams that we update all existing accounts.

Reply:
ok

Reply:
yes, we are traversing a list of accounts it is not required, updating. thanks